src/DynamoUtilities/CLIWrapper.cs

```diff
@@ -162,6 +165,47 @@ protected virtual string GetData(int timeoutms, Func<string> mockReadLine = null
             return completedTask == readStdOutTask ? readStdOutTask.Result : string.Empty;
         }
 
+        private void DecodeHTMLContent(ref string htmlContent)
+        {
+            //Usually the response from Md2Html.exe are in the format <<<<< response >>>>> so we avoid decoding those strings
+            if (!htmlContent.Contains("<<<<<"))
+            {
+                //Decode HTML File paragraphs
+                DecodeBase64(ref htmlContent, @"<p>(.*?)</p>");
+
+                //Decode HTML File headers
+                DecodeBase64(ref htmlContent, @"<h2\s.*>(.*?)</h2>");
+            }
+        }
+
+        private void DecodeBase64(ref string htmlContent, string regExp)
+        {
+            if (!string.IsNullOrEmpty(htmlContent))
+            {
+                //TODO- missing to support other HTML tags
+                var Matches = Regex.Matches(htmlContent, regExp, RegexOptions.IgnoreCase);
+                string encodedString = string.Empty;
+                if (Matches.Count > 0)
+                {
+                    //Add validation is groups is null
+                    encodedString = Matches[0].Groups[1].Value;
+                    if (!string.IsNullOrEmpty(encodedString) && !encodedString.Trim().StartsWith("<img"))
+                    {
+                        //Due that in some cases there are nested tags inside <p> or <h2> and if the info is not encoded then it will send an exception so we will leave it as it is
+                        try
+                        {
+                            var base64Line = Convert.FromBase64String(encodedString);
+                            var decodedString = Encoding.UTF8.GetString(base64Line);
+                            htmlContent = htmlContent.Replace(encodedString, decodedString);
+                        }
+                        catch (Exception ex)
```

Comment:
You can do `catch (Exception)` if you dont plan to print out ex info

Reply:
Updated the catch in the next commit: https://github.com/DynamoDS/Dynamo/pull/15377/commits/33430d9cb836caa94d02429978d28021579d2319